toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1005 @@
+    </message>
+    <message>
+        <source>Show Mesh</source>
+        <translation>Mesh zeigen</translation>
+    </message>
+    <message>
+        <source>Show Rigidity</source>
+        <translation>Starrheit zeigen</translation>
+    </message>
+    <message>
+        <source>Show SO</source>
+        <translation>SO zeigen</translation>
+    </message>
+    <message>
+        <source>Show Skeleton Onion Skin</source>
+        <translation>Skelett Onion Skin zeigen</translation>
+    </message>
+    <message>
+        <source>The previous vertex name will be discarded, and all associated keys will be lost.
+
+Do you want to proceed?</source>
+        <translation>Der vorherige Vertex-Name wird verworfen, und all zugewiesenen Tasten gehen verloren.
+
+Möchten Sie fortfahren?</translation>
+    </message>
+    <message>
+        <source>Edit Mesh</source>
+        <translation>Mesh editieren</translation>
+    </message>
+    <message>
+        <source>Snap To Mesh</source>
+        <translation>An Mesh einrasten</translation>
+    </message>
+    <message>
+        <source>Angle Bounds</source>
+        <translation>Winkelbegrenzung</translation>
+    </message>
+    <message>
+        <source>Swap Edge</source>
+        <translation>Kanten tauschen</translation>
+    </message>
+    <message>
+        <source>Collapse Edge</source>
+        <translation>Kante löschen</translation>
+    </message>
+    <message>
+        <source>Split Edge</source>
+        <translation>Kante teilen</translation>
+    </message>
+    <message>
+        <source>Cut Mesh</source>
+        <translation>Mesh schneiden</translation>
+    </message>
+</context>
+<context>
+    <name>PlasticToolOptionsBox</name>
+    <message>
+        <source>Create Mesh</source>
+        <translation>Mesh erschaffen</translation>
+    </message>
+    <message>
+        <source>Skeleton:</source>
+        <translation>Skelett</translation>
+    </message>
+    <message>
+        <source>SO</source>
+        <translation>SO</translation>
+    </message>
+    <message>
+        <source>Angle</source>
+        <translation>Winkel</translation>
+    </message>
+    <message>
+        <source>Distance</source>
+        <translation>Abstand</translation>
+    </message>
+</context>
+<context>
+    <name>PrimitiveParam</name>
+    <message>
+        <source>Shape:</source>
+        <translation>Form:</translation>
+    </message>
+    <message>
+        <source>Thickness:</source>
+        <translation>Dicke：</translation>
+    </message>
+    <message>
+        <source>Opacity:</source>
+        <translation>Opazität:</translation>
+    </message>
+    <message>
+        <source>Hardness:</source>
+        <translation>Härte:</translation>
+    </message>
+    <message>
+        <source>Polygon Sides:</source>
+        <translation>Polygon-Seiten:</translation>
+    </message>
+    <message>
+        <source>Auto Group</source>
+        <translation>Automatische Gruppe</translation>
```

Comment:
Use "Automatisch gruppieren" or "Automatisches Gruppieren" to refer to the process of grouping instead of the result.